### PR TITLE
Wlt 694 split createmember

### PR DESCRIPTION
**Pull request by pol9kov**
**- What I did**
Split createMember to contract.createMember and migration.createMember
**- How I did it**
Split createMember to contract.createMember and migration.createMember.

**- How to verify it**
Launch tests in contract_create_member_test.go and migration_create_member_test.go
**- Description for the changelog**

### CODE REVIEW

File: logicrunner/builtin/contract/member/member.go [hunk 1]
```diff
@@ -316,17 +324,25 @@ func (m *Member) getNodeRef(publicKey string) (interface{}, error) {
 	return nodeRef, nil
 }
 
+type ContractCreateMemberOutput struct {
```
Comment (LinnikD) [inline logicrunner/builtin/contract/member/member.go]: ContractCreateMemberOutput -> CreateResponse

File: logicrunner/builtin/contract/member/member.go [hunk 2]
```diff
@@ -316,17 +324,25 @@ func (m *Member) getNodeRef(publicKey string) (interface{}, error) {
...
+	Reference string `json:"reference"`
+}
+type MigrationCreateMemberOutput struct {
```
Comment (LinnikD) [inline logicrunner/builtin/contract/member/member.go]: MigrationCreateMemberOutput  -> MigrationCreateResponse

File: logicrunner/builtin/contract/member/member.go (hunk 1 above)
Comment (pol9kov) [reply logicrunner/builtin/contract/member/member.go]: Maybe like this ContractCreateMemberOutput -> ContractCreateMemberResponse?

Comment (LinnikD) [reply logicrunner/builtin/contract/member/member.go]: yes, its ok too

**Pull request merged**